client/hotrod-client/src/test/java/org/infinispan/client/hotrod/multimap/RemoteMultimapStateTransferTest.java

```diff
@@ -27,13 +30,46 @@ public class RemoteMultimapStateTransferTest extends MultiHotRodServersTest {
    private static final int VALUES = 4;
    private static final String CACHE_NAME = "multimap";
 
+   private ClientIntelligence clientIntelligence;
+
+   public RemoteMultimapStateTransferTest clientIntelligence(ClientIntelligence clientIntelligence) {
+      this.clientIntelligence = clientIntelligence;
+      return this;
+   }
+
+   @Override
+   public Object[] factory() {
+      return Arrays.stream(ClientIntelligence.values())
+            .flatMap(ci -> Stream.of(CacheMode.DIST_SYNC, CacheMode.DIST_ASYNC)
+                  .map(cm -> new RemoteMultimapStateTransferTest().clientIntelligence(ci).cacheMode(cm)))
+            .toArray();
+   }
+
+   @Override
+   protected Object[] parameterValues() {
+      return concat(super.parameterValues(), clientIntelligence);
+   }
+
+   @Override
+   protected String[] parameterNames() {
+      return concat(super.parameterNames(), "clientIntelligence");
+   }
+
    @Override
    protected void createCacheManagers() throws Throwable {
-      ConfigurationBuilder cacheBuilder = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false);
+      ConfigurationBuilder cacheBuilder = getDefaultClusteredCacheConfig(CacheMode.DIST_ASYNC, false);
```

Comment:
Oops this should be `cacheMode` and not `CacheMode.DIST_ASYNC`. I will submit a follow up PR.